### PR TITLE
lualatex: add page

### PR DESCRIPTION
**Pull request by narula2000**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 1.15.0

### CODE REVIEW

File: pages/common/lualatex.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# lualatex
+
+> An extended version of TeX using Lua to compile.
+> More information: <https://manned.org/lualatex.1>.
+
+- Start `texlua` it act as Lua interpreter:
```
Comment (kbdharun) [inline pages/common/lualatex.md:6]:
```suggestion
- Start `texlua` to act as Lua interpreter:
```

File: pages/common/lualatex.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Compile Tex file to PDF:
+
+`lualatex {{path/to/file.tex}}`
+
+- Compile Tex file without error interruption:
+
+`lualatex -interaction nonstopmode {{path/to/file.tex}}`
+
+- Compile Tex file with specific output file name:
```
Comment (kbdharun) [inline pages/common/lualatex.md:18]:
```suggestion
- Compile Tex file with a specific output file name:
```

File: pages/common/lualatex.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# lualatex
+
+> An extended version of TeX using Lua to compile.
+> More information: <https://manned.org/lualatex.1>.
+
+- Start `texlua` to act as Lua interpreter:
+
+`lualatex`
+
+- Compile Tex file to PDF:
```
Comment (CleanMachine1) [inline pages/common/lualatex.md:10]:
```suggestion
- Compile a Tex file to PDF:
```

File: pages/common/lualatex.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`lualatex {{path/to/file.tex}}`
+
+- Compile Tex file without error interruption:
```
Comment (CleanMachine1) [inline pages/common/lualatex.md:14]:
```suggestion
- Compile a Tex file without error interruption:
```

File: pages/common/lualatex.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`lualatex -interaction nonstopmode {{path/to/file.tex}}`
+
+- Compile Tex file with a specific output file name:
```
Comment (CleanMachine1) [inline pages/common/lualatex.md:18]:
```suggestion
- Compile a Tex file with a specific output file name:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution @narula2000.

File: pages/common/lualatex.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# lualatex
+
+> An extended version of TeX using Lua to compile.
+> More information: <https://manned.org/lualatex.1>.
+
+- Start `texlua` to act as Lua interpreter:
```
Comment (CleanMachine1) [inline pages/common/lualatex.md:6]:
```suggestion
- Start `texlua` to act as a Lua interpreter:
```

last suggestions from me, then I approve.

**Review by CleanMachine1 — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**